modules/support-collecting-host-network-trace.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * support/gathering-cluster-data.adoc
+
+[id="support-collecting-host-network-trace_{context}"]
+= Collecting a host network trace
+
+Sometimes, troubleshooting a network-related issue is simplified by tracing network communication and capturing packets on multiple nodes at the same time.
+
+You can use a combination of the `oc adm must-gather` command and the `registry.redhat.io/openshift4/network-tools-rhel8` container image to gather packet captures from nodes.
+You can analyze the packet captures to understand network communication and assist with diagnosing network issues.
+
+The `oc adm must-gather` command is used to run the `tcpdump` command in pods on specific nodes.
+The `tcpdump` command records the packet captures in the pods.
+When the `tcpdump` command exits, the `oc adm must-gather` command transfers the files with the packet captures from the pods to your client machine.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Start a packet capture from the host network on some nodes:
++
+[source,terminal]
+----
+$ oc adm must-gather \
+    --dest-dir /tmp/captures \  <.>
+    --source-dir '/tmp/tcpdump/' \  <.>
+    --image registry.redhat.io/openshift4/network-tools-rhel8:latest \  <.>
+    --node-selector 'node-role.kubernetes.io/worker' \  <.>
+    --host-network \  <.>
+    -- \
+    timeout 30 tcpdump \  <.>
+    -i any \  <.>
+    -w /tmp/tcpdump/\$POD_NAME-%Y-%m-%dT%H:%M:%S.pcap -W 1 -G 300  <.>
+----
+<.> The command stores the packet captures in directories that are relative to `/tmp/captures` on the client machine. You can specify any writable directory.
+<.> When `tcpdump` is run in the debug pod that `oc adm must-gather` starts, the packet captures are temporarily stored in the `/tmp/tcpdump` directory on the pod.
+<.> The specified image includes the `tcpdump` command that is needed to perform the packet captures.
+<.> The packet captures are performed on worker nodes because they match the specified node selector. As an alternative, you can specify the `--node-name` argument instead to run the packet capture on a single node. If you omit both the `--node-selector` and the `--node-name` argument, the packet captures are performed on all nodes.
+<.> The `--host-network` argument is required so that the packet captures are performed on the network interfaces of the node.
+<.> The `tcpdump` command is run with a timeout of 30 seconds. The timeout is optional.
```

Comment:
Hmm.  I think `tcpdump` will run forever if the `timeout` command is not used. I'll indicate that too: <s>The timeout is optional.</s>If you do not specify `timeout` and a duration, the `tcpdump` command runs forever."

Reply:
@mikemckiernan: looks like default is 10 mins: 
https://github.com/openshift/network-tools/blob/bcfec9c5a054f4c60356175b42a2477ec181fbd9/docs/user.md?plain=1#L206

could you please correct this?

Reply:
Super interesting.  The example from Alex that I started with used the `timeout` command that is part of the OS, not the `--timeout` argument that is part of `oc adm must-gather` (with the 10 minute limit).

I revised the command to use the `--timeout` argument rather than the `timeout` command,  and updated the description to include mention of the 10 minute limit as a default.  Thanks for drawing my attention to this oversight of mine.